src/ray/raylet/main.cc

```diff
@@ -82,22 +80,12 @@ int main(int argc, char *argv[]) {
   const std::string redis_password = FLAGS_redis_password;
   const std::string temp_dir = FLAGS_temp_dir;
   const std::string session_dir = FLAGS_session_dir;
-  const bool disable_stats = FLAGS_disable_stats;
-  const std::string stat_address = FLAGS_stat_address;
```

Comment:
Do we still need to be able to specify an address here as well as the port?

Reply:
Currently, we only export metrics to the local reporter, so we don't need address. We will need it once we want to support configurable cpp exporters (Ant's requirement).